dev/io.openliberty.restfulWS.3.0_fat/fat/src/io/openliberty/restfulWS30/fat/validator/ValidatorTestServlet.java

```diff
@@ -0,0 +1,97 @@
+/*******************************************************************************
+ * Copyright (c) 2020 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package io.openliberty.restfulWS30.fat.validator;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import java.io.InputStream;
+import java.net.HttpURLConnection;
+import java.net.URI;
+import java.util.HashSet;
+import java.util.Set;
+
+import jakarta.servlet.annotation.WebServlet;
+import jakarta.validation.ConstraintViolation;
+import jakarta.validation.Validation;
+import jakarta.validation.Validator;
+import jakarta.validation.constraints.NotNull;
+
+import org.junit.Test;
+
+import componenttest.app.FATServlet;
+
+@SuppressWarnings("serial")
+@WebServlet("/ValidatorTestServlet")
+public class ValidatorTestServlet extends FATServlet {
+
+    @Test
+    public void testNotNullViolations() {
+        Validator validator = Validation.buildDefaultValidatorFactory().getValidator();
+        System.out.println("testNotNullViolations validator " + validator.toString());
+        
+        MyBean myBean = new MyBean();
+        myBean.notNull = null;
+        assertViolations(validator.validate(myBean), NotNull.class);
+    }
```

Comment:
I don't think we need this test.  It's basically just testing manual bean validation - I think the BV FATs have something similar.  We really only need tests that bean validation is performed on JAX-RS resources and providers.